commands.txt

```diff
@@ -4,5 +4,5 @@ export SD_CURL_CMD_WRAPPER="sd-step exec --pkg-version 7.54.1 core/curl"
 export SD_UNZIP_CMD_WRAPPER="sd-step exec --pkg-version 6.0 core/unzip"
 $SD_CURL_CMD_WRAPPER "curl -o ./sonarscanner.zip -L https://sonarsource.bintray.com/Distribution/sonar-scanner-cli/sonar-scanner-cli-3.1.0.1141-linux.zip"
 $SD_UNZIP_CMD_WRAPPER "unzip -q ./sonarscanner.zip"
-SONAR_TOKEN=`$SD_CURL_CMD_WRAPPER "curl -H \"Authorization: Bearer $SD_TOKEN\" $SD_API_AUTH_URL"`
-sonar-scanner-3.1.0.1141-linux/bin/sonar-scanner -Dsonar.host.url=$SONAR_HOST -Dsonar.login=$SONAR_TOKEN -Dsonar.projectKey="job:$SD_JOB_ID" -Dsonar.projectName="$SD_PIPELINE_NAME:$SD_JOB_NAME" -Dsonar.projectVersion=$SD_BUILD_SHA $SD_SONAR_OPTS
+SONAR_TOKEN=`$SD_CURL_CMD_WRAPPER "curl -H \"Authorization: Bearer $SD_TOKEN\" $SD_COVERAGE_AUTH_URL"`
+sonar-scanner-3.1.0.1141-linux/bin/sonar-scanner -Dsonar.host.url=$SD_COVERAGE_HOST -Dsonar.login=$SONAR_TOKEN -Dsonar.projectKey="job:$SD_JOB_ID" -Dsonar.projectName="$SD_PIPELINE_NAME:$SD_JOB_NAME" -Dsonar.projectVersion=$SD_BUILD_SHA $SD_SONAR_OPTS
```

Comment:
`SD_SONAR_HOST` makes more sense than `SD_COVERAGE_HOST`, since this won't work against any COVERAGE HOST it has to be Sonar.  
Also what if we try out a different service for Coverage and we want both services to be active.

Reply:
With current coverage plugin implementation, a cluster can only have one coverage plugin. If we want multiple coverage services, then we need a router and `SD_COVERAGE_AUTH_URL` is going to be different for each service. With a single plugin available in the system, `SD_COVERAGE_HOST` makes more sense.

Reply:
@jithin1987 are you okay with 
```
["SD_SONAR_HOST": "https://sonar.screwdriver.cd", "SD_SONAR_AUTH_URL": "https://api.screwdriver.cd/v4/coverage/token"]
```

Reply:
are you talking from API auth standpoint ? Whether we do multiple or not since we are specifically dealing with Sonar I think it still makes sens to call it `SD_SONAR_HOST`

Reply:
>  are you okay with SD_SONAR_HOST and SD_SONAR_AUTH_URL?

Yep :)